frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -1,504 +1,14 @@
-import * as _ from 'lodash';
-import {
-  DashboardsOverviewHealthResourceSubsystem,
-  HorizontalNavTab,
-  ModelDefinition,
-  ModelFeatureFlag,
-  Plugin,
-  ResourceTabPage,
-  RoutePage,
-  ResourceDetailsPage,
-  DashboardsOverviewResourceActivity,
-  CustomFeatureFlag,
-  ProjectDashboardInventoryItem,
-  ResourceListPage,
-} from '@console/plugin-sdk';
-import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
-import { referenceForModel } from '@console/internal/module/k8s';
 import * as models from './models';
-import * as mockModels from './mock-models';
-import { getCephHealthState } from './components/dashboards/persistent-internal/status-card/utils';
-import { isClusterExpandActivity } from './components/dashboards/persistent-internal/activity-card/cluster-expand-activity';
-import { WatchCephResource } from './types';
-import {
-  detectOCS,
-  detectOCSSupportedFeatures,
-  detectRGW,
-  CEPH_FLAG,
-  MCG_FLAG,
-  OCS_FLAG,
-  ODF_MANAGED_FLAG,
-  detectManagedODF,
-  detectComponents,
-  FEATURES,
-  RGW_FLAG,
-  ODF_ADMIN,
-} from './features';
 import { ODF_MODEL_FLAG } from './constants';
-import { getObcStatusGroups } from './components/dashboards/object-service/buckets-card/utils';
 
-type ConsumedExtensions =
-  | ModelFeatureFlag
-  | HorizontalNavTab
-  | ModelDefinition
-  | DashboardsOverviewHealthResourceSubsystem<WatchCephResource>
-  | RoutePage
-  | CustomFeatureFlag
-  | ResourceDetailsPage
-  | ResourceTabPage
-  | DashboardsOverviewResourceActivity
-  | ProjectDashboardInventoryItem
-  | ResourceListPage;
-
-const apiObjectRef = referenceForModel(models.OCSServiceModel);
-const blockPoolRef = referenceForModel(models.CephBlockPoolModel);
-const storageSystemGvk = referenceForModel(models.StorageSystemModel);
-
-const OCS_MODEL_FLAG = 'OCS_MODEL';
-
-const plugin: Plugin<ConsumedExtensions> = [
-  {
-    type: 'ModelDefinition',
-    properties: {
-      models: _.values(models),
-    },
-  },
```

Comment:
we need this... we are not setting this in odf-console...
please refer to previous comments... we only need to remove those extensions for which we do not have corresponding dynamic extensions added in odf-console...
please just don;t remove anything without checking throughly...

Reply:
Ok Sure, I do not see any failure after i remove this with our without ODF installed.